backend/src/main/java/com/github/boltzmann/biokiste/backend/repository/BoxRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.github.boltzmann.biokiste.backend.repository;
+
+import com.github.boltzmann.biokiste.backend.model.OrganicBox;
+import org.springframework.data.mongodb.repository.MongoRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface BoxRepository extends MongoRepository<OrganicBox, String> {
+}
```

Comment:
u.c. you could rename the class to OrganicBoxRepository to keep the naming consistent

Reply:
For what do you use this repo? As far as I can see, you store the boxes as part of the AppUserDetails. Will you store the data separetly in future?

Reply:
Will do.